modules/logging-loki-cli-install.adoc

```diff
@@ -18,7 +18,39 @@ To install and configure logging on your {product-title} cluster, additional Ope
 
 .Procedure
 
-. Create a `Subscription` object:
+--
+include::snippets/logging-stable-updates-snip.adoc[leveloffset=+1]
+--
+
+. Apply the `openshift-operators-redhat` namespace object YAML configuration that follows:
++
+[source,terminal]
+----
+$ oc apply -f openshift_operators_redhat_namespace.yaml
+----
++
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-operators-redhat <1>
+  annotations:
+    openshift.io/node-selector: ""
+  labels:
+    openshift.io/cluster-monitoring: "true" <2>
+----
+<1> You must specify the `openshift-operators-redhat` namespace. To prevent possible conflicts with metrics, you should configure the Prometheus Cluster Monitoring stack to scrape metrics from the `openshift-operators-redhat` namespace and not the `openshift-operators` namespace. The `openshift-operators` namespace might contain community Operators, which are untrusted and could publish a metric with the same name as an {product-title} metric, which would cause conflicts.
+<2> String. Enables metric service discovery.
+--
+
+.  Apply the Subscription object YAML configuration that follows:
```

Comment:
Same sentiments here as above: why not have the YAML first, then the command to create it? Or cat EOF? 

Also, all commands should have enter/run the following command. see: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command